Makefile.devcontainer

```diff
@@ -65,10 +65,11 @@ fuzz-grafana:                   ## Run fuzzer for services/grafana package.
 check:                          ## Run required checkers and linters.
 	go run .github/check-license.go
 	go-sumtype ./...
-	golangci-lint run -c=.golangci-required.yml
 
 check-all: check                ## Run all linters for new code.
 	golangci-lint run -c=.golangci.yml --new-from-rev=main
+	golangci-lint run -c=.golangci-required.yml
```

Comment:
Shouldn't we run `.golangci-required.yml` inside `check` command as told in command's description?

Reply:
In [CI workflow](https://github.com/percona/pmm-managed/blob/2bb2fd1b55ce8c9004e11a23d9100e0cd0f2b48c/.github/workflows/ci.yml#L186), we run `make check` and `golangci-lint` separately. Leaving it inside `check` means it will run twice and cause CI to fail because of timeout occasionally.

I guess we should update `check` description at least anyway.

Reply:
Let's do it in a different way: just drop `make check` from CI configuration and invoke all required tools directly there. And `make check` we will keep for manual usage.